src/neml2/drivers/Driver.cxx

```diff
@@ -44,6 +46,9 @@ Driver::expected_options()
   options.set<bool>("verbose") = false;
   options.set("verbose").doc() = "Whether to output additional logging information";
 
+  options.set<Size>("random_seed") = -1;
```

Comment:
Why not just the usual check to see if the user set it?  Why use the magic number?

Reply:
Good point.